src/compiler/core.ts

```diff
@@ -2901,6 +2901,13 @@ namespace ts {
             return value;
         }
 
+        export function assertEachDefined<T, A extends ReadonlyArray<T>>(value: A): A {

```

Comment:
Add a required message tag here as well.. it will be much simpler to understand what is going on with a message